operations/template/key.tf

```diff
@@ -28,6 +28,17 @@ resource "azurerm_key_vault_access_policy" "allow_github_deployer" {
     "Delete",
     "Purge",
   ]
```

Comment:
In the similar TI PR, `Recover` was added to the `secret_permissions` - does that need to happen here?

Reply:
I don't think we need `Recover`, That may have been some thrashing that we missed in TI